backend/controllers/projects.go

```
@@ -1439,3 +1453,90 @@
+		prBatches, err := models.DB.GetDiggerBatchesForPR(batch.RepoFullName, batch.PrNumber)
+		if err != nil {
+			slog.Error("Error getting PR service",
+				"batchId", batch.ID,
+				"error", err,
+			)
+			return fmt.Errorf("error getting github service: %v", err)
+		}
+
+		for _, prBatch := range prBatches {
+			jobs, err := models.DB.GetDiggerJobsForBatch(prBatch.ID)
+			if err != nil {
```

golangci-lint (1.64.8)

SA9003: empty branch

(staticcheck)